dapp/src/components/dialogs/CreateSubnameDialog.tsx

```diff
@@ -19,14 +19,63 @@ import {
     LoadingIndicator,
 } from '@iota/apps-ui-kit';
 import { useCurrentAccount, useIotaClient, useSignAndExecuteTransaction } from '@iota/dapp-kit';
-import { isSubname, MIN_LABEL_SIZE } from '@iota/iota-names-sdk';
+import { isSubname, MIN_LABEL_SIZE, NameRecord } from '@iota/iota-names-sdk';
 import { useMutation, useQueryClient } from '@tanstack/react-query';
 import { ChangeEvent, useState } from 'react';
 
 import { NameRecordData, queryKey, useNameRecord, useRegistrationNfts } from '@/hooks';
 import { NameUpdate, useUpdateNameTransaction } from '@/hooks/useUpdateNameTransaction';
+import { RegistrationNft } from '@/lib/interfaces';
 import { getNameObject, isNameRecordExpired } from '@/lib/utils/names';
 
+function createCreateSubnameUpdates({
```

Comment:
create createe :notes: 
```suggestion
function createSubnameUpdates({
```